app/questionnaire/placeholder_parser.py

```diff
@@ -95,7 +99,8 @@ def __call__(
                 routing_path_block_ids_map
             )
             self._value_source_resolver = self._get_value_source_resolver(
-                routing_path_block_ids
+                routing_path_block_ids=routing_path_block_ids,
+                assess_routing_path=assess_routing_path,
```

Comment:
This is setting assess true for the main resolver that is used by multiple transforms even the ones that dont use the transform we care about and the ones that shouldn't use the path. You need to get a value source resolver based on the transform at the point of resolution. It works for calc summary because, calc summary here sets assess false and in Value source resolver, it will set to True when needed.

For yours, you need to create a new instance of VSE with assess True when it is a transform that require path. You need to do it at the `for transform in transform_list:` level.

Reply:
Made some changes. Is this what you were meaning?